swift/Tests/Blockchains/BluzelleTests.swift

```diff
@@ -0,0 +1,210 @@
+                  "type": "tendermint/PubKeySecp256k1",
+                  "value": "AlcobsPzfTNVe7uqAAsndErJAjqplnyudaGB0f+R+p3F"
+                },
+                "signature": "gKWKk5lku9H/wosvnsD6auWUt7AGNjLEudWS+Fdv4TQr16HUBVpkq9A6gZYqZIawFukq2I/qrtyw9MKR4pVkhg=="
+              }
+            ]
+          }
+        }
+        """
+
+        let actualSignedJson = AnySigner.signJSON(inputJson, key: privateKeyData, coin: .bluzelle)
+        
+        XCTAssertJSONEqual(expectedSignedJson, actualSignedJson)
+
+    }
+
+    func testSigningMessage() {
+        let privateKey = PrivateKey(data: privateKeyData)!
+
+        let sendCoinsMessage = CosmosMessage.Send.with {
+            $0.fromAddress = myAddress
+            $0.toAddress = "bluzelle1xccvees6ev4wm2r49rc6ptulsdxa8x8jfpmund"
+            $0.amounts = [CosmosAmount.with {
+                $0.amount = 1
+                $0.denom = "ubnt"
+            }]
+        }
+
+        let message = CosmosMessage.with {
+            $0.sendCoinsMessage = sendCoinsMessage
+        }
+
+        let fee = CosmosFee.with {
+            $0.gas = 200000
+            $0.amounts = [CosmosAmount.with {
+                $0.amount = 200
+                $0.denom = "ubnt"
+            }]
+        }
+
+        let input = CosmosSigningInput.with {
+            $0.accountNumber = 1037
+            $0.chainID = "bluzelle"
+            $0.memo = ""
+            $0.sequence = 8
+            $0.messages = [message]
+            $0.fee = fee
+            $0.privateKey = privateKey.data
+        }
+
+        let output: CosmosSigningOutput = AnySigner.sign(input: input, coin: .bluzelle)
+
+        let expectedJSON: String =
+        """
+        {
+          "mode": "block",
+          "tx": {
+            "fee": {
+              "amount": [
+                {
+                  "amount": "200",
+                  "denom": "ubnt"
+                }
+              ],
+              "gas": "200000"
+            },
+            "memo": "",
+            "msg": [
+              {
+                "type": "cosmos-sdk/MsgSend",
+                "value": {
+                  "amount": [
+                    {
+                      "amount": "1",
+                      "denom": "ubnt"
+                    }
+                  ],
+                  "from_address": "bluzelle1hsk6jryyqjfhp5dhc55tc9jtckygx0epzzw0fm",
+                  "to_address": "bluzelle1xccvees6ev4wm2r49rc6ptulsdxa8x8jfpmund"
+                }
+              }
+            ],
+            "signatures": [
+              {
+                "pub_key": {
+                  "type": "tendermint/PubKeySecp256k1",
+                  "value": "AlcobsPzfTNVe7uqAAsndErJAjqplnyudaGB0f+R+p3F"
+                },
+                "signature": "0StN9LHr4fIbEzCzyx5bzWJmfgiUqkwoavYGNUDm2shXKmAFogVtOviC4zMNNHKBtFTnHQ07DO1UjqEEWS/2BA=="
+              }
+            ]
+          }
+        }
+        """
+
+        XCTAssertJSONEqual(expectedJSON, output.json)
```

Comment:
This is a real world tx?

Reply:
No this is a simulated transaction. For a real world tx

chain-id: net-6
account_number: 590

Has this to be realworld?

Reply:
yes, to make sure the code is working

Reply:
This is fixed.